viewing_party/party.py

```diff
@@ -1,23 +1,213 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+from symbol import single_input
+
+
+def create_movie(title, genre, rating): #Wave 1, step 1 done
+    movie_dict={}
+    if title is None or genre is None or rating is None:
+        return None
+
+    movie_dict["title"]=title
+    movie_dict["genre"]=genre
+    movie_dict["rating"]=rating
+    return movie_dict
+
+def add_to_watched(user_watched_dict,movie_info):
+    user_watched_dict["watched"] = [movie_info]
+    return user_watched_dict
+
+def add_to_watchlist(user_watchlist_dict, movie_info):
+    user_watchlist_dict["watchlist"] = [movie_info]
+    return user_watchlist_dict
+
+def watch_movie(single_user_data,movie_title):    
+    list_of_watchlist_movie_info=single_user_data["watchlist"]
+    list_of_watched_movie_info=single_user_data["watched"]
+
+    #Adds movie info to list of watched movies if it is in the watchlist and then removes the movie info from the watchlist
+    for index,movie_info in enumerate(list_of_watchlist_movie_info):
+        if movie_title == movie_info["title"]:
+            list_of_watched_movie_info.append(movie_info)
+            list_of_watchlist_movie_info.pop(index)

```

Comment:
nicely done! I see you used `enumerate` in order to access index position and the movie dictionary more easily. 

We could use `remove` method instead of `pop` if we wanted to use a simpler for loop, which takes a value as a parameter instead of an index position.

Your solution works just fine as is!